tests/contenttypes_tests/models.py

```diff
@@ -55,6 +55,21 @@ class Meta:
         proxy = True
 
 
+class GenericConcreteModel(models.Model):
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    content_object = GenericForeignKey()
```

Comment:
Please reuse the `Answer` model below by defining an `AnswerProxy` model instead. We try to avoid adding a models if possible as they require the creation an extra database table on every test run.